### PR TITLE
Update speech support

### PR DESCRIPTION
**Pull request by dpvc**
This PR moves the `aria-label` attributes to the `mjx-container` rather than the element corresponding to the math element, and makes the internal svg `title` element optional (so that it can be included in conversions from tex to svg, for example, where the `mjx-container` is not retained.  This is controlled by the `internalSpeechTitles` option for the SVG output jax.

It also adds `role="application"` when the explorer is present (should this always be set when speech is attached?), and adds `role="img" focusable="false"` to svg elements, as in v2.

### CODE REVIEW

File: mathjax3-ts/a11y/semantic-enrich.ts [hunk 1]
```diff
@@ -118,6 +128,24 @@ export function EnrichedMathItemMixin<N, T, D, B extends Constructor<AbstractMat
             this.state(STATE.ENRICHED);
         }
 
+        /**
+         * @param {MathDocument} docuemnt   The MathDocument for the MathItem
```
[the remark this answers is not in the document]
Comment (dpvc) [reply mathjax3-ts/a11y/semantic-enrich.ts]: OOPS!, sorry, merged to soon.  I'll make this correction in master.
